tests/test_rooms/test_create_room.py

```diff
@@ -25,7 +24,7 @@ def test_room_creation(self):
         """
         Testing for room creation
         """
-        headers = {"Authorization": "Bearer" + " " + admin_api_token}
+        headers = {"Authorization": "Bearer" + " " + os.getenv('ADMIN_TOKEN')}
```

Comment:
Would it be nice to keep the token fixtures file, and have variables such ADMIN_TOKEN = os.getenv('ADMIN_TOKEN') instead of getting it from env file in all places.

Reply:
I had done that at first, I then found that the instances that we used the tokens were few and thought that the current approach was easier. I will update this right away.